Tests/SWBBuildSystemTests/ShellScriptSandboxingTests.swift

```diff
@@ -642,14 +642,14 @@ fileprivate struct ShellScriptSandboxingTests: CoreBasedTests {
                 }
 
                 // Change $A_DOT_TXT_PREFIX
-                overridenParameters = BuildParameters(configuration: "Debug",
-                                                      overrides: ["ENABLE_USER_SCRIPT_SANDBOXING": enableSandboxingInTest,
-                                                                  "A_DOT_TXT_PREFIX": "UPDATED_A_PREFIX",
-                                                                  "B_DOT_TXT_PREFIX": "B_PREFIX",
-                                                                 ])
+                overriddenParameters = BuildParameters(configuration: "Debug",
+                                                       overrides: ["ENABLE_USER_SCRIPT_SANDBOXING": enableSandboxingInTest,
+                                                                   "A_DOT_TXT_PREFIX": "UPDATED_A_PREFIX",
+                                                                   "B_DOT_TXT_PREFIX": "B_PREFIX",
+                                                                  ])
```

Comment:
Are these formatting changes necessary in this PR? Just asking

Reply:
The spelling correction changed the length of the word by one character and therefore required re-aligning the following lines to start the first character of the parameter label with the opening parenthesis per usual Swift style.